src/main/scala/rocket/DCache.scala

```diff
@@ -557,13 +557,15 @@ class DCacheModule(outer: DCache) extends HellaCacheModule(outer) {
     val user_bit_cacheable = edge.manager.supportsAcquireTFast(access_address, a_size)
```

Comment:
The calculation of 'user_bit_cacheable' is potentially expensive. Can we reuse this calculation from the TLB permissions check? @aswaterman

Reply:
For rocket/bullet/mallard, yeah we may be able to. Caboose would have to do something else.